Lib/test/test_capi/test_long.py

```diff
@@ -643,6 +643,51 @@ def test_long_getsign(self):
 
         # CRASHES getsign(NULL)
 
+    def test_long_ispositive(self):
+        # Test PyLong_IsPositive()
+        ispositive = _testcapi.pylong_ispositive
+        self.assertEqual(ispositive(1), 1)
+        self.assertEqual(ispositive(123), 1)
+        self.assertEqual(ispositive(-1), 0)
+        self.assertEqual(ispositive(0), 0)
+        self.assertEqual(ispositive(True), 1)
+        self.assertEqual(ispositive(False), 0)
+        self.assertEqual(ispositive(IntSubclass(-1)), 0)
+        self.assertRaises(TypeError, ispositive, 1.0)
+        self.assertRaises(TypeError, ispositive, Index(123))
+
+        # CRACHES ispositive(NULL)
```

Comment:
Doesn't seem to crash with `ispositive(NULL)`, right? If so, the same change could be applied to `isnegative`and `iszero`.

```suggestion
        self.assertRaises(TypeError, ispositive, NULL)
```